### DISCUSSION TITLE
Custom pattern clozes

### DISCUSSION THREAD

**Discussion by talacchi** [st3v3nmw/obsidian-spaced-repetition#1176] [Q&A]
Hello,

I am trying to make flashcards where a same card would hide many answers, looking like so:

Option one : [...], then [...]
Option two : [...](hint)

which would then show :
Option one : [answer 1], then [answer 2]
Option two : [answer 3]

I've tried follow the tips on this [page](https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/cloze-cards/), but each cloze generates a card, which ends up being quickly overwhelming. How am I supposed to write this for it too appear formatted on my cards?

I use curly braket to generate the clozes. I love the custom pattern ``{{[123::]answer[::hint]}}``, except I already used ``::`` in too many cards to change it. Is there a way to use a custom format like ``{{[123\\]answer[\\hint]}}`` for example ?
Can I get an explanation about how to use the 123 ?
Thank you very much !

**Comment by oumout** [comment]
Hi,

lots of tips [here](https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/cloze-cards/#cloze-types) about how to use 123's..
